backend/infrahub/core/diff/calculator.py

```diff
@@ -18,6 +18,7 @@ async def calculate_diff(
         diff_branch: Branch,
         from_time: Timestamp,
         to_time: Timestamp,
+        is_first_diff: bool = False,
```

Comment:
the `is_first_diff` and `include_unchanged` args are used to keep UNCHANGED values around until a little later in the diff incremental calculation process where they are used for setting previous values